### PR TITLE
Bukkit

### PR DESCRIPTION
**Pull request by Azim-D3Tm**
Added Bukkit support:
Now bukkit plugins can use TelegramBotsAPI !

### CODE REVIEW

File: telegrambots/pom.xml [hunk 1]
```diff
@@ -80,8 +80,25 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
-
+      <repositories>
+    <repository>
+        <id>spigot-repo</id>
+        <url>https://hub.spigotmc.org/nexus/content/repositories/snapshots/</url>
```
Comment (Azim-D3Tm) [inline telegrambots/pom.xml]: Spigot repository. Can be replaced with bukkit, but imho spigot will live longer

File: telegrambots/pom.xml [hunk 2]
```diff
@@ -80,8 +80,25 @@
...
+    </repository>
+  </repositories>
     <dependencies>
+      <dependency>
+  		<groupId>org.bukkit</groupId>
+        <artifactId>bukkit</artifactId>
+        <version>1.12.1-R0.1-SNAPSHOT</version>
```
Comment (Azim-D3Tm) [inline telegrambots/pom.xml]: latest bukkit version, becouse it's backward-compatible

File: telegrambots/pom.xml [hunk 3]
```diff
@@ -184,6 +201,16 @@
         <finalName>${project.artifactId}-${project.version}</finalName>
         <testOutputDirectory>${project.build.directory}/test-classes</testOutputDirectory>
         <sourceDirectory>${project.basedir}/src/main/java</sourceDirectory>
+        <resources>
+            <resource>
+            <targetPath>.</targetPath>
+            <filtering>true</filtering>
+            <directory>${basedir}</directory>
+                <includes>
+                    <include>plugin.yml</include>
```
Comment (Azim-D3Tm) [inline telegrambots/pom.xml]: This file lets bukkit know the head class of plugins

**Comment by rubenlagus** [comment]
@Azim-D3Tm This look like too specific to be added to an standard library, sorry

**Pull request closed without merging**